homework-service/HWS/HWS.Dal.Sql/Groups/GroupRepository.cs

```diff
@@ -92,6 +94,26 @@ public async Task<Domain.Group> FindById(Guid id)
             return group;
         }
 
+        public async Task<Domain.Comment> InsertComment(Domain.User user, Domain.Group group, string content)
+        {
+            var dbComment = new GroupComment
+            {
+                _id = 0,
+                Id = Guid.NewGuid(),
+                CreatedBy = user.UserFullName,
+                GroupId = group._id,
+                CreationDate = DateTime.Now,
+                Content = content,
+            };
+
+            _comments.Add(dbComment);
```

Comment:
Bevallom én a házi létrehozásánál azért a group-ot mentettem el, mert fordítva a navigation property-k miatt azt hitte, hogy a csoportot is most hoznám létre. Itt viszont nincs ilyen probléma. úgy hogy elhiszem, hogy működhet.